platform/linux-generic/pktio/dpdk.c

```diff
@@ -1249,10 +1253,16 @@ static int dpdk_open(odp_pktio_t id ODP_UNUSED,
 		pkt_dpdk->vdev_sysc_promisc = 1;
 	rte_eth_promiscuous_disable(pkt_dpdk->port_id);
 
-	if (!strcmp(dev_info.driver_name, "rte_ixgbe_pmd"))
-		pkt_dpdk->min_rx_burst = DPDK_IXGBE_MIN_RX_BURST;
+	/* Drivers requiring minimum burst size. Supports also *_vf versions
+	 * of the drivers. */
+	if (!strncmp(dev_info.driver_name, IXGBE_DRV_NAME,
+		     strlen(IXGBE_DRV_NAME)) ||
+	    !strncmp(dev_info.driver_name, I40E_DRV_NAME,
+		     strlen(I40E_DRV_NAME)))
+		pkt_dpdk->min_rx_burst = DPDK_MIN_RX_BURST;
 	else
 		pkt_dpdk->min_rx_burst = 0;
+
```

Comment:
Wouldn't it be simpler (and more extensible) to have `min_rx_burst` as a field in the `dev_info` struct?